docs/source/item-validation.rst

```diff
@@ -188,7 +192,11 @@ as a `dict`:
 
     # settings.py
 
-    SPIDERMON_VALIDATION_SCHEMAS: {
+    from tutorial.items import DummyItem, OtherItem
+
+    ...
```

Comment:
Can you build the docs, to check if this separation looks good?
I feel it would be fine without it, as most people have the habit of copy-pasting code snippets without thinking much. In that it scenario it would lead to errors.

Reply:
hooo, i see, i just updated this one